client/my-sites/importer/newsletter/subscribers/step-initial.tsx

```diff
@@ -37,34 +38,42 @@ export default function StepInitial( {
 		prevInProgress.current = importSelector?.inProgress;
 	}, [ importSelector?.inProgress, setAutoFetchData ] );
 
+	const isOnFreePlan = useSelector( ( state ) => isSiteOnFreePlan( state, selectedSite.ID ) );
+
 	return (
 		<Card>
 			<h2>{ translate( 'Step 1: Export your subscribers from Substack' ) }</h2>
 			<p>
 				{ translate(
-					'Generate a CSV of your Substack subscribers. In Substack, go to {{strong}}Subscribers{{/strong}}, click {{strong}}Export{{/strong}} under "All subscribers," then upload the CSV in the next step. On the free plan, {{supportLink}}you can import up to 100 subscribers.{{/supportLink}}',
+					'In Substack, go to {{strong}}Subscribers{{/strong}}, scroll down to the "All subscribers" table, click the … menu on the right and select Export. Choose the "Export all columns" option and download the CSV file. Then upload the CSV below.',
```

Comment:
The `Export` at the end of the first sentences was strong before, maybe we should make this and the `...` strong as well?